SECURITY.md

```diff
@@ -0,0 +1,93 @@
+# Security Policy
+
+1. [Reporting security problems](#reporting)
+4. [Security Bug Bounties](#bounty)
+2. [Incident Response Process](#process)
+
+<a name="reporting"></a>
+## Reporting security problems to Solana
+
+**DO NOT CREATE AN ISSUE** to report a security problem. Instead, please send an
+email to security@solana.com and provide your github username so we can add you
+to a new draft security advisory for further discussion.
+
+Expect a response as fast as possible, within one business day at the latest.
+
+<a name="bounty"></a>
+## Security Bug Bounties
+We offer bounties for critical security issues.  Please see [Bug Bounty
+Compensation](https://forums.solana.com/t/tour-de-sol-updates-to-tour-de-sol-and-bug-bounty-compensation-structure/1132)
+for more details.
+
+<a name="process"></a>
+## Incident Response Process
+
+In case an incident is discovered or reported, the following process will be
+followed to contain, respond and remediate:
+
+### 1. Establish a new draft security advisory
+In response to an email to security@solana.com, a project administrator will create a
+new draft security advisory for the incident at
+https://github.com/solana-labs/solana/security/advisories with the available
+information.
+
+By default only the core development team and incident reporter will have access
```

Comment:
We should probably create a smaller incident response GH team that has default access to advisory creation/admin and adds relevant folks as needed. Probably the same folks who will be replying to security@solana.com.

Current defaults aren't great.  AFAICT every Solana employee with a github account, including c-suite and non-eng has access.  That's a lot of potentially compromised accounts.

Reply:
Done!  So we have two groups now:
* @solana-labs/admins - this is a highly restricted group that gets to create new security advisories based on email to security(at)solana.com.
* @solana-labs/security-incident-response - our first wave of responders for new incidents, this group is added by an admin when the new draft security advisory is created.   This group should have access to pull in other folks from @solana-labs/core-contributors or elsewhere as needed

Reply:
Perfect!